presto-druid/pom.xml

```diff
@@ -245,6 +245,7 @@
             <artifactId>validation-api</artifactId>
         </dependency>
 
+        <!-- This is needed as a return value from and argument to the Druid API. -->
```

Comment:
you can remove this or add a TODO.

Reply:
you can mention in a TODO that remove the joda dependency once druid stops depending on joda.